CONTRIBUTING.md

```diff
@@ -1,5 +1,32 @@
 # Contributing to Blockly
 
-Please make pull requests against develop, not master.  If your patch needs to go into master immediately, include a note in your PR.
+Want to contribute? Great!
+- First, read this page (including the small print at the end).
+- Second, please make pull requests against develop, not master. If your patch
+  needs to go into master immediately, include a note in your PR.
 
-For more information, head over to the [Blockly Developers site](https://developers.google.com/blockly/guides/modify/contributing).
+For more information on style guide and other details, head over to the [Blockly Developers site](https://developers.google.com/blockly/guides/modify/contributing).
+
+### Before you contribute
+Before we can use your code, you must sign the
+[Google Individual Contributor License Agreement](https://cla.developers.google.com/about/google-individual)
+(CLA), which you can do online. The CLA is necessary mainly because you own the
+copyright to your changes, even after your contribution becomes part of our
+codebase, so we need your permission to use and distribute your code. We also
+need to be sure of various other things—for instance that you'll tell us if you
+know that your code infringes on other people's patents. You don't have to sign
+the CLA until after you've submitted your code for review and a member has
+approved it, but you must do it before we can put your code into our codebase.
+Before you start working on a larger contribution, you should get in touch with
```

Comment:
This should be a new paragraph--it's not related to the CLA.

Maybe also note that there are plenty of things tagged "help wanted" on the issue tracker page, if someone just wants to get their feet wet with open source.